tools/makeboards.py

```diff
@@ -97,12 +113,13 @@ def MakeBoard(name, prettyname, pid, boarddefine, flashsizemb, boot2):
         BuildFreq(n)
         BuildDebugPort(n)
         BuildDebugLevel(n)
+        BuildUSBStack(n)
         if name == "generic":
             BuildBoot(n)
 
 
 BuildGlobalMenuList()
-MakeBoard("rpipico", "Raspberry Pi Pico", "0x000a", "RASPBERRY_PI_PICO", 2, "boot2_w25q080_2_padded_checksum")
-MakeBoard("adafruitfeather", "Adafruit Feather RP2040", "0x000b", "ADAFRUIT_FEATHER_RP2040", 8, "boot2_w25x10cl_4_padded_checksum")
-MakeBoard("generic", "Generic RP2040", "0xf00a", "GENERIC_RP2040", 16, "boot2_generic_03h_4_padded_checksum")
+MakeBoard("rpipico", "Raspberry Pi", "Pico", "0x2e8a", "0x000a", "RASPBERRY_PI_PICO", 2, "boot2_w25q080_2_padded_checksum")
+MakeBoard("adafruitfeather", "Adafruit", "Feather RP2040", "0x239a", "0x80f1", "ADAFRUIT_FEATHER_RP2040", 8, "boot2_w25x10cl_4_padded_checksum")
```

Comment:
Adafruit Feather RP2040 has its own dedicated VID/PID for use with Arduino, we should use it instead of

![image](https://user-images.githubusercontent.com/249515/118632353-85d57d00-b7fa-11eb-934d-da7b750f2795.png)

Reply:
Awesome, thanks!  I just made up the non-RPI USB IDs since when I was doing this and not aware of their official ones.